src/main/java/bookface/logic/commands/add/AddBookCommand.java

```diff
@@ -0,0 +1,46 @@
+package bookface.logic.commands.add;
+
+import static bookface.logic.parser.CliSyntax.PREFIX_AUTHOR;
+import static bookface.logic.parser.CliSyntax.PREFIX_TITLE;
+import static java.util.Objects.requireNonNull;
+
+import bookface.logic.commands.CommandResult;
+import bookface.logic.commands.exceptions.CommandException;
+import bookface.model.Model;
+import bookface.model.book.Book;
+
+/**
+ * The command that adds a book to the BookList.
+ */
+public class AddBookCommand extends AddCommand {
+    public static final String COMMAND_WORD = "book";
+    public static final String MESSAGE_USAGE = AddCommand.COMMAND_WORD
+            + " " + COMMAND_WORD
+            + ": Adds a book to the book list."
+            + "Parameters: "
+            + PREFIX_AUTHOR + "AUTHOR "
+            + PREFIX_TITLE + "TITLE \n"
```

Comment:
Perhaps the order of the parameters should be changed here to match the order of the command fields as well?

Reply:
my bad, fixed now